var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -62,6 +62,21 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
 
     depends_on("cmake@3.18:", type="build")
 
+    # Profiling
+    depends_on(
+        "hiop+deepchecking build_type=RelWithDebInfo", when="+hiop build_type=RelWithDebInfo"
+    )
+    depends_on("hiop~deepchecking  build_type=Release ", when="+hiop build_type=Release ")
+
+    # Control the package's build-type depending on the release or debug flag
+    for pkg in ["raja", "umpire", "magma", "camp"]:
```

Comment:
```suggestion
    for pkg in ["raja", "umpire", "magma", "gingko"]:
```

Reply:
Just one minor change, and then I think this is ready to merge. Thanks @ryandanehy!!!